<!-- Core Tools Policy Translator -->
<!--
  Author: Ian Loefgren
  Last Modified: 5/24/2017

  The ROS launch file for the Core Tools policy translator ROS service.
-->

<launch>

  <!-- Policy Translator -->
  <node name="policy_translator_server" pkg="policy_translator"
    type="policy_translator_server.py" output="screen"/>

  <!-- Belief map image publishing -->
  <node name="interface_map" pkg="policy_translator"
    type="Image_pub.py" output="screen"/>

</launch>
